src/views/Activity.vue

```diff
@@ -174,6 +174,8 @@ div
       div
         b-form-checkbox(v-model="filterAFK")
           | Filter away AFK time
+        b-form-checkbox(v-model="countAudibleAsActive")
```

Comment:
Nice!

Reply:
Thinking of removing this though, just adds unnecessary complexity. It's not like I'm expecting anyone to not want this, and it isn't saved across page reloads anyway which makes it kinda bad UX.

Reply:
It is important if someone for example often leaves their computer for extended periods of time with for example Spotify or YouTube in the foreground while not actually being at the computer but only having it for playing music.